app/jobs/account_creation_threat_metrix_job.rb

```diff
@@ -6,13 +6,15 @@ def perform(
     threatmetrix_session_id: nil,
     request_ip: nil,
     email: nil,
-    uuid_prefix: nil
+    uuid_prefix: nil,
+    uuid: nil
```

Comment:
This will need to be a separate pull request / separate deploy, per [50/50 deploy guidance on adding a field to an existing job](https://handbook.login.gov/articles/manage-50-50-state.html#add-an-argument-to-a-job-perform-method).

Reply:
oh good point. will do.